src/Modules/Shared/PSReadLine/PSReadLine.psd1

```diff
@@ -13,5 +13,6 @@ CLRVersion = '4.0'
 FunctionsToExport = 'PSConsoleHostReadline'
 CmdletsToExport = 'Get-PSReadlineKeyHandler','Set-PSReadlineKeyHandler','Remove-PSReadlineKeyHandler',
                   'Get-PSReadlineOption','Set-PSReadlineOption'
-HelpInfoURI = 'https://go.microsoft.com/fwlink/?LinkId=528806'
+HelpInfoURI = 'https://go.microsoft.com/fwlink/?linkid=855966
+'
```

Comment:
Is this quote put on a new line intentionally?

Reply:
Fixed.